src/progress_bar.rs

```diff
@@ -489,6 +490,34 @@ impl ProgressBar {
     }
 }
 
+/// The alignment value for `MultiProgress`, the alignment controls how the multi progress
```

Comment:
Thanks for the great docs here! They should start with a single line that has a short one-line explanation, then an empty line, then the longer explanation.